contextproject-ms3/src/main/java/contextproject/models/Track.java

```diff
@@ -0,0 +1,104 @@
+package contextproject.models;
+
+import com.mpatric.mp3agic.InvalidDataException;
+import com.mpatric.mp3agic.Mp3File;
+import com.mpatric.mp3agic.UnsupportedTagException;
+
+import java.io.IOException;
+
+public class Track {
+  private Mp3File song;
+  private String title;
+  private String artist;
+  private String album;
+  private String absolutepath;
+  private long length;
+
+  /**
+   * constructor of a track.
+   * 
+   * @param abPath
+   *          Path of the mp3 file
+   */
+  public Track(String abPath) {
+    
+    try {
+      song = new Mp3File(abPath);
+    } catch (UnsupportedTagException e) {
+      System.out.println("There was a Unsupported tag exception with file:" + abPath);
```

Comment:
I think we can later implement a logger instead of directly printing to sysout. For now it's okay though. :)